client/styles/components/_toolbar.scss

```diff
@@ -103,6 +103,7 @@
 
 .toolbar__project-owner {
 	margin-left: #{5 / $base-font-size}rem;
+	color: #bfbfbf;
```

Comment:
Because the editor has different themes, you need to use a SCSS mixin/map to set colors so that they are correct in all of the themes. This style should be
```scss
.toolbar__project-owner {
  @include themify() {
    margin-left: #{5 / $base-font-size}rem;
    color: getThemifyVariable('inactive-text-color');
  }
}
```
if you look at `client/styles/base/_base.scss`, you'll see that all `<a>` tags have this color.